lib/mandrill-transport.js

```diff
@@ -37,16 +37,16 @@ MandrillTransport.prototype.send = function(mail, callback) {
       text: data.text,
       html: data.html
     }
-  }, function(result) {
-    var response = result[0] || {};
-
-    if (['sent', 'queued'].indexOf(response.status) <= -1) {
-      var err = new Error(response.reject_reason || 'unsent');
-      err.result = result;
-      callback(err);
-    } else {
-      callback(null, { messageId: response._id });
-    }
+  }, function(results) {
+
+    var info = {accepted: [], rejected: []};
+    results.forEach(function(result) {
+      if (['sent', 'queued', 'scheduled'].indexOf(result.status) !== -1)
+        info.accepted.push(result);
+      else
+        info.rejected.push(result);
+    });
```

Comment:
This looks right, but I'm looking into Mandrill's responses in more detail.

But, I think we should be using braces with multiline blocks - https://github.com/RebelMail/javascript#blocks